gflownet/proxy/crystals.py

```diff
@@ -0,0 +1,111 @@
+from gflownet.proxy.base import Proxy
+import torch
+import torch.nn as nn
+
+import os.path as osp
+
+
+class SendekMLPWrapper(Proxy):
+    """
+    Wrapper for MLP proxy trained on Li-ion SSB ionic conductivities calculated
+    from Sendek et. al's logistic regression model
+
+    Attributes
+    ----------
+
+    feature_set: str
+        supports either "comp" or "all", with the former including only
+        features that denote the composition of a crystal in the form of
+        [Li_content, Natoms, e1, e2, e3...]. The latter contains one-hot
+        encoded space-group values as well as the crystal structure
+        paramaters [a, b, c, alpha, beta, gamma]
+
+    path_to_proxy: str
+        path to saved torch checkpoint for MLP state dictionary
+
+    scale: bool
+        whether or not the input nereds to be standardised based on
+        mean and standard deviation of the dataset
+    """
+
+    def __init__(self, feature_set, path_to_proxy, scale=False, **kwargs):
+        super().__init__(**kwargs)
+        # TODO: assert oracle_split in ["D2_target", "D2_target_fid1", "D2_target_fid2"]
+        # TODO: assert oracle_type in ["MLP"]
+        if feature_set == "comp":
+            self.oracle = CrystalMLP(85, [256, 256])
+        elif feature_set == "all":
+            self.oracle = CrystalMLP(322, [512, 1024, 1024, 512])
+        self.oracle.load_state_dict(
+            torch.load(osp.join(path_to_proxy, feature_set + ".ckpt"))
+        )
+        self.oracle.to(self.device)
+        if scale:
+            self.scale = {
+                "mean": torch.load(osp.join(path_to_proxy, feature_set + "_mean.pt")),
+                "std": torch.load(osp.join(path_to_proxy, feature_set + "_std.pt")),
+            }
+        else:
+            self.scale = None
+
+    def __call__(self, crystals):
+        """
+        Returns a vector of size [batch_size] that are calculated
+        ionic conductivity values between 0 and 1
+        """
+
+        if self.scale is not None:
+            crystals = (crystals - self.scale["mean"]) / self.scale["std"]
+        crystals = torch.nan_to_num(crystals, nan=0.0)
+        with torch.no_grad():
+            scaled_ionic_conductivity = self.oracle(crystals.to(torch.float32))
```

Comment:
```suggestion
            scaled_ionic_conductivity = self.oracle(crystals.to(device=self.device, dtype=self.float))
```
The class proxy has attributes device and float to be used for this :)